client/src/components/App/Content/useContent.ts

```diff
@@ -0,0 +1,49 @@
+import Swal from 'sweetalert2';
+import theme from 'styles/theme';
+
+import useStyles from './ContentStyles';
+import { useContentOutcome } from './ContentInterfaces';
+
+const useContent = (): useContentOutcome => {
```

Comment:
should be useForm for example adn not useContext, because content will later have a landing page as well.
maybe move content of Content to Form component